Framework/Lumberjack/CocoaLumberjack.h

```diff
@@ -0,0 +1,39 @@
+//
+//  CocoaLumberjack.h
+//  CocoaLumberjack
+//
+//  Created by Andrew Mackenzie-Ross on 3/02/2015.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+//! Project version number for CocoaLumberjack.
+FOUNDATION_EXPORT double CocoaLumberjackVersionNumber;
+
+//! Project version string for CocoaLumberjack.
+FOUNDATION_EXPORT const unsigned char CocoaLumberjackVersionString[];
```

Comment:
Not sure about what prevents it from being a shared header between CocoaPods, manual importas and framework/Carthage. What is Imports instead of ""?